subsys/mpsl/pm/Kconfig

```
@@ -15,6 +15,19 @@ config MPSL_USE_ZEPHYR_PM
 
 if MPSL_USE_ZEPHYR_PM
 
+config MPSL_PM_UNINIT_WORK_WAIT_TIME_MS
+	int "Time the MPSL PM uninit procedure will wait for work queue to complete in milliseconds"
+	default 100
+	help
+	  The option provides a timeout value the MPLS PM uninit procedure will wait
```

maybe mention that the value is in ms again?
```suggestion
	  The option specifies a timeout value in milliseconds the MPLS PM uninit procedure will wait
```